api.Dockerfile

```diff
@@ -0,0 +1,42 @@
+# First step: build the assets
+FROM node:lts-alpine AS builder
+
+WORKDIR /
+ADD package.json yarn.lock ./
+ADD ./apis/core/package.json ./apis/core/
+ADD ./ui/package.json ./ui/
```

Comment:
Do we need the UI package.json here?

Reply:
Strictly it would not be but that we cannot simply `COPY . .` and selectively choose the files needed for build. It does not hurt and the Dockerfiles is a little simpler